c2corg_ui/static/js/apiservice.js

```diff
@@ -380,6 +380,20 @@ app.Api.prototype.updatePreferredLanguage = function(lang) {
 
 
 /**
+ * @param {string} url
+ * @return {!angular.$q.Promise<!angular.$http.Response>}
+ */
+app.Api.prototype.readFeed = function(url) {
```

Comment:
Wouldn't it be more consistent to have the URL defined in this service (as for the other functions) and only pass a "type" parameter (standard, personal, profile) as argument?